README.md

```diff
@@ -129,6 +129,27 @@ spec:
     namespace: flux-system
 ```
 
+### Gzip Encode Terraform Plan
+
+Due to the size limitation of a Kubernetes `Secret` of 1 MB, gzip is useful to compress the outputted terraform plan:
+
+```yaml
+apiVersion: infra.contrib.fluxcd.io/v1alpha1
+kind: Terraform
+metadata:
+  name: helloworld
+  namespace: flux-system
+  annotations:
```

Comment:
Sorry for not being clear.

Encoding should be transparent to users. What I meant by having "encoding": "gzip" was that the annotation present in the tfplan Secrets to tell that their byte arrays are being encoded by gzip.